### PR TITLE
reset blink when cursor updates

### PR DESCRIPTION
**Pull request by AntonBogun**

### CODE REVIEW

**Comment by AntonBogun** [comment]
Is the review being held "pending"?

**Comment by aardappel** [comment]
I am not sure what you are asking.. there are review comments to address?

**Comment by aardappel** [comment]
Argh.. github's UI makes it really easy to have unsubmitted comments and then never see that they're unsubmitted.

File: src/document.h [hunk 1]
```diff
@@ -322,18 +322,28 @@ struct Document {
         }
     }
 
-    void Blink() {
+    void HandleBlink(bool resetBlink = false) {
```
[the remark this answers is not in the document]
Comment (AntonBogun) [reply src/document.h:325]: Hmmm I will try, though I presume there is no place where they'd be all neatly written in right?

[the remark this answers is not in the document]
Comment (aardappel) [reply src/document.h]: Just look at the surrounding code for style

**Pull request merged**